AngularApp/src/app/diagnostic-data/components/detector-view/detector-view.component.ts

```diff
@@ -56,6 +57,12 @@ export class DetectorViewComponent implements OnInit {
         this.ratingEventProperties = {
           "DetectorId": data.metadata.id
         }
+      
+        let authors = data.metadata.author.split(",");
```

Comment:
Also add ; as a separator.. I saw few folks using , and few using ;
So maybe include common separators like , : ; and space

Reply:
Sure. Let me include those separators as well:)